kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/strimzi/Strimzi.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.installation.strimzi;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.kroxylicious.systemtests.Constants;
+import io.kroxylicious.systemtests.utils.DeploymentUtils;
+
+import static io.kroxylicious.systemtests.k8s.KubeClusterResource.kubeClient;
+
+/**
+ * The type Strimzi.
+ */
+public class Strimzi {
+    private static final Logger LOGGER = LoggerFactory.getLogger(Strimzi.class);
+    private final String deploymentNamespace;
+    private final String installationUrl;
+
+    /**
+     * Instantiates a new Strimzi.
+     *
+     * @param deploymentNamespace the deployment namespace
+     */
+    public Strimzi(String deploymentNamespace) {
+        this.deploymentNamespace = deploymentNamespace;
+        installationUrl = "https://strimzi.io/install/latest?namespace=" + deploymentNamespace;
```

Comment:
Eventually we'd want to substitute an AMQ Streams equivalent, so the url probably needs to come from an env var.

Reply:
done